README.md

```diff
@@ -172,7 +172,7 @@ Feel free to check the [issues page](../../issues/).
 
 > Write a message to encourage readers to support your project
 
-If you like this project...
+Give a star if you like this project!
```

Comment:
- [optional] kindly remove the `live demo` section, since you don't yet have a deployed version of your app. You can add it later on when you have one.